src/test/java/net/masterthought/cucumber/generators/TagsOverviewPageTest.java

```diff
@@ -38,14 +38,14 @@ public void getWebPage_ReturnsTagOverviewReportFileName() {
     public void prepareReport_AddsCustomProperties() {
 
         // given
-        page = new TagsOverviewPage(reportResult, configuration);
+    	VelocityContext context = new VelocityContext();
```

Comment:
here and in other places: this is global or page context as the name does not provide answer for that question